docker/dockerfiles/be/be_entrypoint.sh

```diff
@@ -109,18 +109,37 @@ collect_env_info
 add_self $svc_name || exit $?
 log_stderr "run start_be.sh"
 
+if [[ "$COREDUMP_ENABLED" == "true" ]]; then
+  # start inotifywait loop daemon to monitor core dump generation
+  $STARROCKS_ROOT/upload_coredump.sh &
+fi
+
+
 addition_args=
 if [[ "x$LOG_CONSOLE" == "x1" ]] ; then
     # env var `LOG_CONSOLE=1` can be added to enable logging to console
     addition_args="--logconsole"
 fi
-$STARROCKS_HOME/bin/start_be.sh $addition_args
-ret=$?
-if [[ $ret -ne 0 && "x$LOG_CONSOLE" != "x1" ]] ; then
-    nol=50
-    log_stderr "Last $nol lines of be.INFO ..."
-    tail -n $nol $STARROCKS_HOME/log/be.INFO
-    log_stderr "Last $nol lines of be.out ..."
-    tail -n $nol $STARROCKS_HOME/log/be.out
-fi
-exit $ret
+
+while true; do
```

Comment:
It's an infinite loop if start successfully?

Reply:
no, if start successfully, it will stuck at https://github.com/StarRocks/starrocks/pull/42883/files#diff-df6bcdc0849060472b310e2597c625213106ea0679d62e7aa9e43c5d7a9e2220R125 until it fails.

This change has been tested internally in our production k8s for several weeks already.